tools/encyclopedia/encyclopedia_fasta_to_prosit_csv.xml

```diff
@@ -0,0 +1,61 @@
+<tool id="encyclopedia_fasta_to_prosit_csv" name="ConvertFastaToPrositCSV" version="@VERSION@.0" python_template_version="3.5">
+    <description>Create Prosit CSV Input From a Protein FASTA</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive"><![CDATA[
+        EncyclopeDIA -convert -fastatoprositcsv 
+        -i '$i'
+        -minCharge $minCharge
+        -maxCharge $maxCharge
+        -defaultCharge $defaultCharge
+        -maxMissedCleavage $maxMissedCleavage
+        -minMz $minMz
+        -maxMz $maxMz
+        -defaultNCE $defaultNCE
+        #if $enzyme
+            -enzyme '$enzyme'
+        #end if
+        -o '$output'
+    ]]></command>
+    <inputs>
+        <param argument="-i" type="data" format="fasta" label="Protein Fasta database"/>
+	<param argument="-minCharge" type="integer" value="2" min="1" max="6" label="minCharge" help="(default: 2)"/>
```

Comment:
I would remove the `default:` here as it is the default already in value

Reply:
Done.